test/storage/snapshots_finish_test.rb

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require_relative "../../src/lib/storage/snapshots_finish"
```

Comment:
this should not be needed, simple `require "storage/snapshots_finish"` should be enough